### PR TITLE
Remove handlers crate target.

### PR DESCRIPTION
**Pull request by ry**
- Add build scripts for Rust flatbuffers.
- Rewrites some reply.cc methods in Rust.
- Changes some struct elements to table in msg.fbs (rust flatbuffers
  lacks support currently)
- Renames handlers_test to test_rs.
- This reorg is needed to make progress on the code cache handler.

### CODE REVIEW

File: build_extra/flatbuffers/rust/rust_flatbuffer.gni [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    outputs = [
+      "$out_dir/{{source_name_part}}_generated.rs",
+    ]
+
+    args = [
+      "--rust",
+
+      #"--gen-mutable",
+      #"--keep-prefix",
```
Comment (piscisaureus) [inline build_extra/flatbuffers/rust/rust_flatbuffer.gni]: Why are these commented out? If they are to stay, add a comment why.

File: tools/util.py [hunk 2]
```diff
@@ -55,3 +55,16 @@ def touch(fname):
         os.utime(fname, None)
     else:
         open(fname, 'a').close()
+
+
+# Recursive search for files of certain extensions.
+# (Recusrive glob doesn't exist in python 2.7.)
+def g(directory, *extensions):
```
Comment (piscisaureus) [inline tools/util.py]: This is too terse a name for an exported function. glob_tree?

**Comment by ry** [comment]
> Spelling (same for the commit message)

Done

> This is too tersely named. Just call it glob_tree or something.

Done. Called it `find_exts`.

File: tools/format.py [hunk 3]
```diff
@@ -12,15 +10,19 @@
...
-# TODO(ry) Install clang-format in third_party.
-run(["clang-format", "-i", "-style", "Google"] + glob("src/*.cc") +
-    glob("src/*.h"))
-for fn in ["BUILD.gn", ".gn"] + glob("build_extra/**/*.gn*"):
+
+# TODO(ry) Use third_party/depot_tools/clang-format.
+run(["clang-format", "-i", "-style", "Google"] + find_exts("src", ".cc", ".h"))
+
+for fn in ["BUILD.gn", ".gn"] + find_exts("build_extra", ".gn", ".gni"):
     run(["gn", "format", fn])
```
Comment (piscisaureus) [inline tools/format.py]: Should also use third_party gn

Comment (ry) [reply tools/format.py]: Fixed.

File: build_extra/flatbuffers/rust/rust_flatbuffer.gni (hunk 1 above)
Comment (ry) [reply build_extra/flatbuffers/rust/rust_flatbuffer.gni]: Removed.

**Comment by ry** [comment]
> I hope we can stop checking in msg_generated.ts some day.

#403

**Pull request merged**